go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -770,6 +771,7 @@ func (npwipt *nodePortWatcherIptables) SyncServices(services []interface{}) {
 	for _, chain := range []string{iptableNodePortChain, iptableExternalIPChain} {
 		recreateIPTRules("nat", chain, keepIPTRules)
 	}
+	return nil
```

Comment:
ditto for this function

Reply:
Tracker under: https://bugzilla.redhat.com/show_bug.cgi?id=2081858